tls/s2n_ktls.c

```diff
@@ -86,9 +101,116 @@ S2N_RESULT s2n_ktls_retrieve_file_descriptor(struct s2n_connection *conn, s2n_kt
     return S2N_RESULT_OK;
 }
 
+S2N_RESULT s2n_ktls_retrieve_io_mode(s2n_ktls_mode ktls_mode, int *tls_tx_rx_mode)
```

Comment:
Nit: "retrieve" is a little nonstandard, particularly because you're not really retrieving anything. Why not "s2n_ktls_get_io_mode"? Or "s2n_ktls_mode_to_io_mode"?